config.json

```diff
@@ -378,6 +378,17 @@
         "random"
       ]
     },
+    {
+      "slug": "yacht",
+      "uuid": "8e129e54-6b5c-4696-84bc-588bc1c7f0f7",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Which core exercise do you think should unlock this?

Reply:
An easy answer would be Bob:
- It's quite early on
- There aren't as many exercises unlocked here, so you wouldn't need to redistribute any other exercises unlocked by Bob.